.ncurc.js

```diff
@@ -8,9 +8,9 @@ module.exports = {
 
     // "Must use import to load ES Module"
     // These can be removed once the tests are converted to Typescript
+    'find-up',
```

Comment:
Might be worth switching to [escalade](https://github.com/lukeed/escalade) sync or async (personally I still believe using async will benefit us but anyway). I tried to make the switch but I'm not familiar with TS.

Reply:
I am fine with either find-up or escalade. It is unlikely that I will upgrade all the tests to typescript since there are so many, so if we stay with `find-up` it will be frozen at `5.0.0`.

Reply:
Unfortunately, I can't make it work, but I think it'll benefit us even if we stick with the `escalade/sync`. Just thinking out loud in case you or someone else has time to make the switch later.